packages/next/server/load-components.ts

```diff
@@ -126,20 +130,6 @@ export async function loadComponents(
 
   const { getServerSideProps, getStaticProps, getStaticPaths } = ComponentMod
 
-  let isAppPath = false
```

Comment:
will always pass down the `isAppPath` flag for that route, so we won't need to double check if it's a path containing `server/app`